frame/plasm-rewards/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+[package]
+name = "pallet-plasm-rewards"
+version = "0.1.0"
+authors = [
+    "Takumi Yamashita <takumi@stake.co.jp>",
+    "Aleksandr Krupenkin <aleksandr@stake.co.jp>"
+]
+edition = "2018"
+
+[dependencies]
+serde = { version = "1.0", optional = true }
+codec = { package = "parity-scale-codec", version = "1.2.0", features = ['derive'], default-features = false }
+primitives = { package = 'sp-core', git = 'https://github.com/staketechnologies/substrate.git', branch = 'plasm-master', default_features = false }
```

Comment:
What's about to rework around package aliaces as it was done in substrate codebase?
For example:
```
primitives = { package = 'sp-core', git = 'https://github.com/staketechnologies/substrate.git', branch = 'plasm-master', default_features = false }
```
->
```
sp-core = { git = 'https://github.com/staketechnologies/substrate.git', branch = 'plasm-master', default_features = false }
```